draft-ietf-rtcweb-jsep.xml

```diff
@@ -3929,13 +3920,16 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
             that the JSEP implementation should be sending media
             ("sendonly" for local answers, "recvonly" for remote
             answers, or "sendrecv" for either type of answer),
-            choose the media format to send
-            as the most preferred media format from the remote
-            description that is also present in the answer, as
-            described in
-            <xref target="RFC3264" />, Sections 6.1 and 7, and start
-            transmitting RTP media once the underlying transport layers
-            have been established. If an SSRC has not already been
+            choose the media format to send; for the offerer, this will be
+            the most preferred media format from the answer that is also
+            locally supported, as described in <xref target="RFC3264" />,
+            Section 7; for the answerer, this will be
```

Comment:
OK. I am going to make the text reflect this (that you can send with a codec you didn't put in your SDP), and that will address Eric's concern as well.